src/app/shell/personal-cabinet/parent/children/children.component.html

```diff
@@ -20,7 +20,7 @@
   <ng-container>
     <div class="card-wrapper">
       <app-child-card class="card" *ngFor="let child of (childrenCards$ | async)?.entities" [child]="child"
-        (deleteChild)="onDelete($event)" [applications]="childApplications(applications$ |async, child)">
+        (deleteChild)="onDelete($event)" [applications]="childApplications(applicationCards$ | async, child)">
```

Comment:
this functionality is broken, as I understood it should be entities 
![image](https://user-images.githubusercontent.com/27493959/176368973-8e2f832f-4db4-45d2-ae0d-03b79a4f86bf.png)